internal/factsengine/gatherers/packageversion.go

```diff
@@ -117,7 +117,10 @@ func executeZypperVersionCmpCommand(
 	}
 
 	versionCmpResult := strings.TrimRight(string(zypperOutput), "\n")
-	result, err := strconv.ParseInt(versionCmpResult, 10, 32)
+	outputParts := strings.Split(versionCmpResult, "\n")
+	comparisonResult := outputParts[len(outputParts)-1]
```

Comment:
Just a doubt here: should we guard against `outputParts` being an empty slice? I guess accessing the last element of an empty slice might break something lol. However not sure that could even happen.